test_solo5_spt.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
```

Comment:
May be we can move this test file and test_solo5_hvt.sh to test directory? These tests should be triggered with jenkins as well